packages/itwinui-variables/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@itwin/itwinui-variables",
+  "version": "0.1.0",
+  "author": "Bentley Systems",
+  "license": "MIT",
+  "main": "index.css",
+  "style": "index.css",
+  "files": [
+    "index.css",
+    "LICENSE.md"
```

Comment:
maybe includ CHANGELOG and README straight away? Or would it have errors if files do not exist?

Reply:
README gets included automatically

cant we just leave this until we have a CHANGELOG?

Reply:
Well, just now I remembered, when we have CHANGELOG, might forget :D 
but yeah, as long as we ensure its there at the end result, Im fine